tools/manage.sh

```diff
@@ -157,6 +157,7 @@ default_help() {
   echo -e "  ${BWhite}ruff-check${RST}      ${BCyan}Run Ruff check for the repository${RST}"
   echo -e "  ${BWhite}ruff-fix${RST}        ${BCyan}Run Ruff fix for the repository${RST}"
   echo -e "  ${BWhite}codespell${RST}       ${BCyan}Run codespell check for the repository${RST}"
+  echo -e "  ${BWhite}run${RST}             ${BCyan}Run the repository${RST}"
```

Comment:
???
What "Run the repository" means?

Reply:
I stole that from the Windows implementation in `manage.ps1`. Basically it allow you to run any command line in the context of the repository (`peotry run` underneath).
Maybe something like `Run a command in the repository environement` would make more sense ?

Reply:
I would say it is connected to https://github.com/ynput/ayon-core/pull/897/files#r1776786031 comment. The `run` is actually `poetry run` -> `Run a poetry command in the repository environement`?

Reply:
This is adjusted, thanks for that.